scipy/_lib/_disjoint_set.py

```diff
@@ -210,6 +210,25 @@ def subset(self, x):
             nxt = self._nbrs[nxt]
         return set(result)
 
+    def subset_size(self, x):
```

Comment:
Could also make sense to include this in the examples on how to use the class, i.e. something like

```python
    >>> disjoint_set.subset('a')
    {'a', 3, 'b'}
    >>> disjoint_set.subset_size('a')
    3
```

Reply:
Sure, done.